lib/puppet_x/puppetlabs/powershell/powershell_manager.rb

```diff
@@ -9,6 +9,8 @@ module PowerShell
     class PowerShellManager
       extend FFI::Library if Puppet::Util::Platform.windows?
 
+      attr_reader :default_execute_timeout
```

Comment:
I needed to add an additional parameter to the execute function.  However I needed to give a value as the parameter is not optional.  I needed to expose the default timeout on the powershell manager object so that I could use it in `lib/puppet/provider/exec/powershell.rb`.  So I created a readonly property.  Not sure if this should be in a different commit or PR.  Also note I needed this value in the integration tests too.